.config/nvim/functions.vim

```diff
@@ -59,3 +59,11 @@ function! ToggleGStatus()
     Gstatus
   endif
 endfunction
+
+function! PryRails()
+  if (&ft=='ruby') 
+    execute "normal! o require 'pry'; binding.pry"
+  else 
+    execute "normal! o <% require 'pry'; binding.pry %>"
```

Comment:
Add this erb tag with a `elseif` block (`elseif (&ft=='eruby')`) and let the else block with no actions.
This way the function will affect only ruby and eruby files